integritybackend/actions.py

```diff
@@ -52,25 +52,14 @@ def archive(self, asset_fullpath: str, org_config: dict, collection_id: str):
         asset_helper = AssetHelper(org_config["id"])
         file_util = FileUtil()
 
-        # Pull things out of the config
-        collection_config = next(
-            (c for c in org_config["collections"] if org_config["collections"][c]['conf']['id'] == collection_id), None
+        org_id = org_config["id"]
```

Comment:
I wonder if the func should just ask for `org_id` rather than whole config.

Reply:
If should, but most of the pipeline is all about config. Flagged this in issue to update.